### PR TITLE
Background blur effect demo

### PR DESCRIPTION
**Pull request by ipasanec**
Hi Joas,
hi Vincent,

these are my code changes for the blur effect.
The blur stream is inserted in MediaDevicesManager. 
In order to end all streams and turn off the camera device, I had to dispatch an event in localmedia.

Sincerely yours
Immanuel

### CODE REVIEW

**Comment by PVince81** [comment]
the code looks good so far, would be good to solve the following:
- [ ] fix eslint issues
- [ ] add the library to package.json with npm install, so it also appears in package-lock.json

I've added next steps for integrating this in the related ticket: https://github.com/nextcloud/spreed/issues/3151

File: package.json [hunk 1]
```diff
@@ -17,37 +17,63 @@
...
 		"attachmediastream": "^2.1.0",
+		"babel-eslint": "^10.1.0",
+		"babel-loader": "^8.2.2",
 		"color.js": "^1.2.0",
 		"crypto-js": "^4.0.0",
+		"css-loader": "^4.3.0",
 		"debounce": "^1.2.1",
 		"emoji-regex": "^9.2.2",
 		"escape-html": "^1.0.3",
+		"eslint": "^7.30.0",
```
Comment (PVince81) [inline package.json:48]: You don't need to include eslint and webpack, these are implicitly imported from the existing dev dependencies `@nextcloud/webpack-vue-config` and others. Not sure if this happened on its own while you updated your branch with master ?

You only need to add the deps needed for video processing.

**Review by PVince81 — COMMENTED** [review]
Thanks for the update, see comment about deps

File: package.json (hunk 1 above)
Comment (ipasanec) [reply package.json:48]: Thanks for the advice!
The dependencies got accidentally while trying to get the dev installation to work. Should have used `--no-save`.
I will commit a fix.

**Comment by PVince81** [comment]
obsoleted by https://github.com/nextcloud/spreed/pull/6036

**Pull request closed without merging**